e2e/testcases/admission_test.go

```diff
@@ -141,55 +148,84 @@ metadata:
 }
 
 func TestDisableWebhookConfigurationUpdate(t *testing.T) {
-	webhook := []byte(`
-apiVersion: admissionregistration.k8s.io/v1
-kind: ValidatingWebhookConfiguration
-metadata:
-  annotations:
-    configsync.gke.io/webhook-configuration-update: disabled
-  name: admission-webhook.configsync.gke.io
-  labels:
-    configmanagement.gke.io/system: "true"
-    configmanagement.gke.io/arch: "csmr"
-`)
-
 	nt := nomostest.New(t, nomostesting.DriftControl)
 
+	// Test starts with Admission Webhook already installed
+	nomostest.WaitForWebhookReadiness(nt)
+
 	nt.Must(nt.RootRepos[configsync.RootSyncName].Add("acme/namespaces/hello/ns.yaml", fake.NamespaceObject("hello")))
 	nt.Must(nt.RootRepos[configsync.RootSyncName].CommitAndPush("add Namespace"))
 	if err := nt.WatchForAllSyncs(); err != nil {
 		nt.T.Fatal(err)
 	}
 
-	if err := os.WriteFile(filepath.Join(nt.TmpDir, "webhook.yaml"), webhook, 0644); err != nil {
-		nt.T.Fatalf("failed to create a tmp file %v", err)
+	helloNS := &corev1.Namespace{}
+	if err := nt.KubeClient.Get("hello", "", helloNS); err != nil {
+		nt.T.Fatal(err)
 	}
+	version := helloNS.ResourceVersion
 
-	// Recreate the admission webhook
-	if _, err := nt.Shell.Kubectl("replace", "-f", filepath.Join(nt.TmpDir, "webhook.yaml")); err != nil {
-		nt.T.Fatalf("failed to replace the admission webhook %v", err)
-	}
+	nomostest.StopWebhook(nt)
 
-	// Verify that the webhook is disabled.
-	if _, err := nt.Shell.Kubectl("delete", "ns", "hello"); err != nil {
-		nt.T.Fatalf("failed to run `kubectl delete ns hello` %v", err)
+	tg := taskgroup.New()
+	tg.Go(func() error {
```

Comment:
Why do we need a taskgroup? It only runs a single validation and waits for the validation result.